query/search.go

```diff
@@ -22,6 +22,7 @@ var es710AggregationField = &AggregationFields{
 	ContentTypes:    "type",
 	PopulationTypes: "population_type.agg_key", // agg_key is {key}###{label} so aggregations on unique combinations of these will be obtained
 	Dimensions:      "dimensions.agg_key",      // agg_key is {key}###{label} so aggregations on unique combinations of these will be obtained
+	DatasetIDs:      "dataset_id",
```

Comment:
I could be wrong, but I don't think we need this here, as we aren't doing a count with the ids